src/index.html

```diff
@@ -17,6 +17,81 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#"
+          class="header__img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Moyo header logo"
+          />
+        </a>
+
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. It seems you have a placeholder `href="#"` that should be replaced with a proper link or removed if not necessary.